### ISSUE TITLE
Inventory filtered by category.

### ISSUE THREAD

**Issue by arionkosturi** [jekkos/opensourcepos#322]
Hi. Can you help me to filter the inventory by categories? I've managed to insert a dropdown but can't populate it from mysql.

**Comment by daN4cat** [comment]
I assume this is sorted by now.

**Comment by arionkosturi** [comment]
Nope. Can't do it by myself and till now had no luck finding someone to help me.

**Comment by daN4cat** [comment]
@arionkosturi we are all busy with boostrapUI upgrade. I understand jekkos can develop new features if you pay him a fee.

**Comment by daN4cat** [comment]
@arionkosturi would adding a category column in current inventory do for you or is do you need a dropdown to filter before entering that view?

**Comment by arionkosturi** [comment]
For me its better to filter before entering the view
